<launch>
  <!-- you can load an exisiting tree with <node ... args="tree.bt"> !-->
  <node pkg="odom_visual_tool" type="odom_visualization_node" name="odom_visualization" output="screen">
    <param name="odom_sub_topic_name" type="string" value="/truck/global/odom" />
    <param name="markers_pub_topic_name" value="/truck/markers" />
    <param name="path_pub_topic_name" value="/truck/path" />
    <param name="max_cnt_odom" value="400" />
  </node>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find odom_visual_tool)/config/odom_visualization.rviz" />
</launch>

